### PR TITLE
Supports 1.13.2

### LINKED ISSUE #13: Update for 1.13.1
I dunno why this wasn't put in but 1.13.1 has been released and it's time to update 😁

### PR DESCRIPTION
**Pull request by ncurran02**
This also resolves #13

### CODE REVIEW

File: src/main/java/net/pravian/aero/util/Materials.java [hunk 1]
```diff
@@ -11,56 +11,56 @@
...
-        MATERIALS.add(Material.MELON_STEM.getId());
-        MATERIALS.add(Material.VINE.getId());
-        MATERIALS.add(Material.FENCE_GATE.getId());
-        MATERIALS.add(Material.WATER_LILY.getId());
-        MATERIALS.add(Material.NETHER_WARTS.getId());
-        MATERIALS.add(Material.WATER.getId());
-        MATERIALS.add(Material.LAVA.getId());
+        final HashSet<String> MATERIALS = new HashSet<>();
+        MATERIALS.add(Material.AIR.toString());
+        MATERIALS.add(Material.LEGACY_SAPLING.toString());
```
Comment (JeromSar) [inline src/main/java/net/pravian/aero/util/Materials.java]: Considering Bukkit's `Material` class has changed a bit. Would you please switch to the new items values (i.e. avoid legacy)?

**Review by JeromSar — CHANGES_REQUESTED** [review]
There are truly only minor edits here and there. Most of it seems in good order.

Could you please:
1. Use Google's formatting guidelines? A IntelliJ plugin which does this automatically is: https://plugins.jetbrains.com/plugin/8527-google-java-format
2. Avoid double paragraph tags (`<p>`) in the javadocs?
3. Ensure the class member order is consistent (e.g. static methods after instance methods).

File: src/main/java/net/pravian/aero/util/Materials.java (hunk 1 above)
Comment (Focusvity) [reply src/main/java/net/pravian/aero/util/Materials.java]: Would you prefer if I have 2 sets? One for new items value and one for legacy?

Comment (JeromSar) [reply src/main/java/net/pravian/aero/util/Materials.java]: I don't think we'll need the legacy items. If I'm not mistaken, the legacy items are just here for backward compatibility for some functionality.

**Comment by Focusvity** [comment]
@JeromSar, is there any other changes needed?

**Comment by Focusvity** [comment]
Hey @JeromSar,
Is there anything you need me to change?

**Comment by JeromSar** [comment]
We're up to date to 1.15.2 now. :)

**Pull request closed without merging**